backend/utils/mail.py

```diff
@@ -27,8 +38,7 @@ def send_email(recipient: str, subject: str, body: str):
         server.sendmail(EMAIL_SENDER, recipient, msg.as_string())
         server.quit()
 
-        print(f"✅ Email sent to {recipient} from {EMAIL_SENDER}")
-
+        print(f"Email sent to {recipient} from {EMAIL_SENDER}")
```

Comment:
tuto print ani nemusí byť, nepotrebujeme logovať každý mail ktorý sa odošle, je to zbytočné